tardis/base.py

```diff
@@ -17,6 +17,7 @@ def run_tardis(
     log_level=None,
     specific_log_level=None,
     show_progress_bars=True,
+    display_widget=True,
```

Comment:
I think this should be explicitly "display_logging_widget"

Reply:
Sure, fixed in https://github.com/tardis-sn/tardis/pull/2872/commits/3187dd15f6f3f5a69b62d4387eba6b215ba2ca71